configs/edison/job_script.edison.bash

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash -l
```

Comment:
Note: It may be possible to write job scripts with python instead of bash, but I have no experience doing this.  I could see the advantages in staying pure python but NERSC doesn't give any examples of slurm batch files using anything other than bash.